server/proxy/pf_server.c

```diff
@@ -233,7 +233,6 @@ static DWORD WINAPI pf_server_handle_client(LPVOID arg)
 	pdata->config = client->ContextExtra;
 	config = pdata->config;
 	client->settings->UseMultimon = TRUE;
-	client->settings->RedirectClipboard = config->Clipboard;
 	client->settings->SupportGraphicsPipeline = config->GFX;
 	client->settings->SupportDynamicChannels = TRUE;
 	client->settings->CertificateFile = _strdup("server.crt");
```

Comment:
Missed this, but it might be a good idea to check `shadow` server to have a proper file path appended.